python/src/ComposedDistribution.i

```diff
@@ -25,21 +25,27 @@ def __ComposedDistribution_repr_html(self):
     # Table of marginals
     html += "<table>\n"
     # Header
-    html += "<tr>\n"
-    html += "  <th>Index</th>\n"
-    html += "  <th>Variable</th>\n"
-    html += "  <th>Distribution</th>\n"
-    html += "</tr>\n"
+    html += "  <tr>\n"
+    html += "    <th>Index</th>\n"
+    html += "    <th>Variable</th>\n"
+    html += "    <th>Distribution</th>\n"
+    html += "  </tr>\n"
     # Content
     for i in range(dimension):
-        html += "<tr>\n"
-        html += "  <td>%d</td>\n" % (i)
-        html += "  <td>%s</td>\n" % (description[i])
-        html += "  <td>%s</td>\n" % (collection[i])
-        html += "</tr>\n"
+        html += "  <tr>\n"
+        html += "    <td>%d</td>\n" % (i)
+        html += "    <td>%s</td>\n" % (description[i])
+        html += "    <td>%s</td>\n" % (collection[i])
+        html += "  </tr>\n"
     html += "</table>\n"
 
     return html
 
 ComposedDistribution._repr_html_ = __ComposedDistribution_repr_html
+
+def __ComposedDistribution_repr_markdown(self):
+    """Get Markdown representation."""
+    return self.repr_markdown()
+
+ComposedDistribution._repr_markdown_ = __ComposedDistribution_repr_markdown
```

Comment:
having a repr_markdown at the c++ level is not consistent with `__str__/__repr__`
I proposed previously that it would be named `__repr_markdown__` both in c++ and python

Reply:
OK, I will do that.

Reply:
Fixed at e3a1d1e54406233038c8297ebc47acf90a4c51e7